### ISSUE TITLE
Host ASCIIDoc instead of PDF in repository

### ISSUE THREAD

**Issue by jerstlouis** [opengeospatial/dggs-4-axis-aligned#1]
Can the actual ASCIIDoc be tracked in the repo instead of the PDF?

Auto-generated PDF are better suited for the "Releases" functionality of GitHub and/or the auto-generated drafts at https://docs.ogc.org/DRAFTS/.

This makes it much easier to collaborate (e.g. submitting Pull Requests etc.) and cloning the repo requires to only transfer the small ASCIIDoc and deltas, as opposed to the large binary files that result in huge repos that take long to clone and pull from.

(I realize this may already be on the GitLab, which I had not yet registered to, but wondering why we would have both public and private repositories for this same specification?).

**Comment by rggibb** [comment]
Hi @jerstlouis the asciidoc is being hosted in the parallel OGC gitlab space for access by the SWG members only, this is more for wider access. I know that in the first instance I only sent the announcement email to SWG members, which does seem a tad perverse but it is my intent to open ot to a wider audience very shortly. The intent for the github space is for wider engagement on the content issues. For Parts 2 & 3, where we don't yet have a draft, this will be more obvious. From a SWG perspective this is our first foray into open public discussion, and so we were more comfortable have =ing the distinction between editorial team access (aka the OGC SWG) and wider discussion on direction, scope and getting finer detail finessed and agreed early in the development rather than the traditional waterfall approach.

**Comment by jerstlouis** [comment]
@rggibb Right. The advantage of having the source repo public is that it allows any contributor to directly submit pull requests on the source, which might save a lot of time for editors since they can be merged with a single click instead of having to dig in and make the changes. Still only editors (not necessarily all SWG members) would have "write" access to the repo, so there is still that distinction of the editorial team, who are the only ones who can merge pull requests.

Since the content is made public anyways, I'm not sure what advantage there is to having a distinct private repo (which is probably why most of the new SWGs are opting to have the source repo public).

Since we are planning to have the issues here in this repo, it's also much easier to have everything in the same repository as the issues and commits and pull requests can all be linked to each other.

(this was just intended as a suggestion -- the editorial team can decide what works best)
